pages/pitch-us.vue

```diff
@@ -1,59 +1,83 @@
 <script setup>
-
+import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
+useMeta({
+  meta: [
+    {
+      name: 'theme-color',
+      content: '#90d8fc',
```

Comment:
like on the "the-lab.vue" file, you can import the colors scss module to have access to the colors.

`import colors from "~/assets/scss/colors.module.scss"`

then select the color for the content meta tag like this:

`content: colors.rlLightblue,`